common/src/unifyfs_client_rpcs.h

```diff
@@ -102,9 +116,9 @@ DECLARE_MARGO_RPC_HANDLER(unifyfs_metaget_rpc)
 
 /* unifyfs_sync_rpc (client => server)
  *
- * given app_id and client_id as input, read all write extents
- * from client index in shared memory and insert corresponding
- * key/value pairs into our global metadata */
+ * given a client identified by (app_id, client_id) as input, read the write
+ * extents for one or more of the client's files from the shared memory index
+ * and update the global metadata for the file(s) */
 MERCURY_GEN_PROC(unifyfs_sync_in_t,
                  ((int32_t)(app_id))
                  ((int32_t)(client_id)))
```

Comment:
@MichaelBrim because the sync rpc is per-file basis, can we add a gfid as an additional parameter here?

Reply:
We could, but the gfid can easily be retrieved from the shmem data, so I'm not sure what is the benefit.

Reply:
We can possibly remove the repeated gfid from `unifyfs_index_t`, but maybe we can address later. We might need it back for different sync operations, if ever.

Reply:
I avoided changing the server rpc handler  (`rm_cmd_sync`) mostly to help avoid any conflicts with work going on in margotreenew.